src/System.ComponentModel.Primitives/src/System.ComponentModel.Primitives.builds

```diff
@@ -6,6 +6,12 @@
     <Project Include="System.ComponentModel.Primitives.csproj">
       <TargetGroup>net45</TargetGroup>
     </Project>
+    <Project Include="System.ComponentModel.Primitives.csproj">
```

Comment:
YOu can remove all the builds from here except for the default and the net463 build.